cmd/microcloud/main_init.go

```diff
@@ -154,15 +148,12 @@ func (c *cmdInit) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	cfg := initConfig{
-		bootstrap:       true,
-		setupMany:       true,
-		address:         c.flagAddress,
-		wipeAllDisks:    c.flagWipeAllDisks,
-		encryptAllDisks: c.flagEncryptAllDisks,
```

Comment:
These fields are no longer assigned on the struct, so I think we can remove them.